.github/workflows/_deploy-analytics-platform.yml

```
@@ -0,0 +1,87 @@
+          env
+          echo "ECR_REPO_URI: ${{ inputs.ECR_REPO_URI }}"
+          echo "AWS_REGION: ${{ inputs.AWS_REGION }}"
+        shell: bash
+      - name: Login to Docker Hub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Checkout
+        uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)